DEVELOPMENT.md

```diff
@@ -47,5 +47,5 @@ No change should be approved until there is documentation for it, supplied in an
 We should always be as transparent as possible.  Sometimes there will be discussions that use customer names, sensitive use cases, and so on.  These must be anonymized, discussed in a private repository, or conducted offline.
 
  - Asynchronous discussions should live in the GitHub issues for this project.
- - Realtime discussions should be in a public chat such as IRC or Slack.
+ - Realtime discussions should be in a public chat such as IRC or the #spec channel on the OpenAPI Slack.
```

Comment:
provide links for both - which IRC host/channel? Does any TDC discussion happen on IRC (I know some @swagger stuff goes on there... does that include OpenAPI spec?)

For Slack - include how one gets an invite to the Slack instance.
Without these pointers, this is not "transparent"